src/VisualStudio/IntegrationTest/IntegrationTests/AbstractIdeEditorTest.cs

```diff
@@ -45,6 +43,7 @@ protected AbstractIdeEditorTest(string solutionName)
             {
                 await SolutionExplorer.CreateSolutionAsync(_solutionName);
                 await SolutionExplorer.AddProjectAsync(ProjectName, _projectTemplate, LanguageName);
+                await SolutionExplorer.RestoreNuGetPackagesAsync(ProjectName);
```

Comment:
📝 This was an attempt to address inconsistent state after creating a new .NET Core project, but it doesn't seem sufficient in itself.